src/Orleans/Runtime/OutsideRuntimeClient.cs

```diff
@@ -837,6 +845,25 @@ public void DeactivateOnIdle(ActivationId id)
 
         #endregion
 
+        private void CurrentDomain_DomainUnload(object sender, EventArgs e)
+        {
+            try
+            {
+                logger.Warn(ErrorCode.ProxyClient_AppDomain_Unload, 
```

Comment:
Yes, you are right. It can be changed to Info.

On the other hand, usually, in a correct execution, you will be calling `GrainClient.Unitialize` and inside `GrainClient.Unitialize` we are unsubscribing from the `DomainUnload` event (I added that in this PR too).
So if the event fired, one can think about it as n failure to properly and cleanly shotdwon the client. But it is not an error, since as GrainClient is GCed .NET will get rid of all resources anyway.